index.html

```diff
@@ -3,11 +3,485 @@
+                <div class="member-icon icon-dribbble-2"></div>
+                <div class="member-icon icon-behance-2"></div>
+              </div>
+            </div>
+            <p class="member-name">James David Doe</p>
+            <p class="member-job">Creative Director</p>
+          </li>
+        </ul>
+      </section>
+      <section id="latest-work">
+        <h1 class="section-title">Latest Work</h1>
+        <p class="section-subtitle">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+        </p>
+        <ul class="work-list">
+          <li class="work-item">
+            <div class="work-box">
+              <div class="work-modal">
+                <button class="work-modal-button icon-attachment"></button>
+                <h1>Illustration</h1>
+                <h2>Graphic / Illustration / Poster</h2>
+              </div>
+              <img src="./images/portfolio-img1.jpg" alt="portfolio-1" />
+            </div>
+          </li>
+          <li class="work-item">
+            <div class="work-box">
+              <div class="work-modal">
+                <button class="work-modal-button icon-attachment"></button>
+                <h1>Illustration</h1>
+                <h2>Graphic / Illustration / Poster</h2>
+              </div>
+              <img src="./images/portfolio-img2.jpg" alt="portfolio-2" />
+            </div>
+          </li>
+          <li class="work-item">
+            <div class="work-box">
+              <div class="work-modal">
+                <button class="work-modal-button icon-attachment"></button>
+                <h1>Illustration</h1>
+                <h2>Graphic / Illustration / Poster</h2>
+              </div>
+              <img src="./images/portfolio-img3.jpg" alt="portfolio-3" />
+            </div>
+          </li>
+          <li class="work-item">
+            <div class="work-box">
+              <div class="work-modal">
+                <button class="work-modal-button icon-attachment"></button>
+                <h1>Illustration</h1>
+                <h2>Graphic / Illustration / Poster</h2>
+              </div>
+              <img src="./images/portfolio-img4.jpg" alt="portfolio-4" />
+            </div>
+          </li>
+          <li class="work-item">
+            <div class="work-box">
+              <div class="work-modal">
+                <button class="work-modal-button icon-attachment"></button>
+                <h1>Illustration</h1>
+                <h2>Graphic / Illustration / Poster</h2>
+              </div>
+              <img src="./images/portfolio-img5.jpg" alt="portfolio-5" />
+            </div>
+          </li>
+          <li class="work-item">
+            <div class="work-box">
+              <div class="work-modal">
+                <button class="work-modal-button icon-attachment"></button>
+                <h1>Illustration</h1>
+                <h2>Graphic / Illustration / Poster</h2>
```

Comment:
the header tags must be used as a way to describe the page map
h1
| h2
| | h3
| | h3
| h2
...